sycl/test-e2e/syclcompat/memory/local_memory_ptr_to_integer.cpp

```diff
@@ -0,0 +1,70 @@
+// REQUIRES: cuda
+// RUN:  %{build} -Xsycl-target-backend --cuda-gpu-arch=sm_75 -o %t.out
+// RUN:  %{run} %t.out
+#include <sycl/detail/core.hpp>
+#include <syclcompat.hpp>
```

Comment:
This shouldn't be needed since you are only using the new functionality from `syclcompat/memory.hpp`

Reply:
Done